plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1878,11 +1873,12 @@ protected void testTargetMaxFileSize(int expectedTableWriters)
         Session session = Session.builder(getSession())
                 .setSystemProperty("task_writer_count", "1")
                 .setSystemProperty("scale_writers", "false")
+                .setSystemProperty("redistribute_writes", "false")
```

Comment:
I am not familiar with this test. Just curious if we are not loosing some coverage with this change.